pkg/registry/apis/query/query.go

```diff
@@ -215,6 +217,15 @@ func (r *queryREST) Connect(connectCtx context.Context, name string, _ runtime.O
 	}), nil
 }
 
+func logEmptyRefids(queries []v0alpha1.DataQuery, logger log.Logger) {
+	for _, q := range queries {
+		if q.RefID == "" {
+			logger.Info("empty refid found", "query_count", len(queries))
```

Comment:
does logging the query count mean something important in this context? It wouldn't tell you how many of the queries are empty right? just the number of queriers in a request where one of them was empty. I'm not sure I follow why that is useful?

Reply:
you are completely correct 👍 .

originally i wanted to log something like "3/5" (3empty out of 5). but then i realized that multiple empty refids are not allowed, so that number would always start with `1`. 

with that said, now, thinking back, i think i will still change it to `3/5`, it will be a lot more clear that way, thanks!

Reply:
coukd i ask for a re-review please? thanks!